src/renderer/entities/contact/model/contact-model.ts

```diff
@@ -24,6 +24,14 @@ const updateContactFx = createEffect(async ({ id, ...rest }: Contact): Promise<C
   return { id, ...rest };
 });
 
+const updateContactsFx = createEffect(async (contacts: Contact[]): Promise<Contact[]> => {
+  if (contacts.length === 9) return [];
```

Comment:
why specifically 9? Maybe add a comment or make a const with this number so it will be clear

Reply:
It's typo🙈